tests/phpunit/tests/ajax/CropImage.php

```diff
@@ -0,0 +1,158 @@
+<?php
+
+/**
+ * Admin Ajax functions to be tested.
+ */
+require_once ABSPATH . 'wp-admin/includes/ajax-actions.php';
+
+require_once ABSPATH . 'wp-admin/includes/class-wp-filesystem-base.php';
+require_once ABSPATH . 'wp-admin/includes/class-wp-filesystem-direct.php';
+
+/**
+ * Class for testing ajax crop image functionality.
+ *
+ * @group ajax
+ */
+class Tests_Ajax_CropImage extends WP_Ajax_UnitTestCase {
+
+	/**
+	 * Tests that attachment properties are copied over to the cropped image.
+	 *
+	 * @ticket 37750
+	 *
+	 * @covers ::wp_ajax_crop_image
+	 */
+	public function test_it_copies_metadata_from_original_image() {
+
+		// Become an administrator.
+		$this->_setRole( 'administrator' );
```

Comment:
As both tests use this, I'd suggest moving it to `set_up()` fixture method instead as then the tests are setup and ready to go.

Reply:
Fixed in 412e86234bf99b738912fecd5d84be4897a4eb30